### DISCUSSION TITLE
Send an email to organizer if a new order has been placed

### DISCUSSION THREAD

**Discussion by Konafets** [pretix/pretix#3453] [Q&A]
Hi, 

I am looking for the function to send an email to the organizer of an event in case a new order has been placed. 

Thanks in advance

**Comment by pc-coholic** [comment]
Ever pretix user can set up a set up a set of notifications - one of them is order notifications.

To use this, log into your pretix account, then click on your name or email-addresses in the top right corner in the purple bar. On the user settings page is a button to set up notifications.

Those notifications will then only be sent to the user you have set them up for.

**Reply by Konafets** [reply]
Thanks for your answer. As mentioned [here](https://github.com/pretix/pretix/discussions/3293), this does not work. I used my user address to order the ticket and received my ticket email but not the notification mail.

**Reply by raphaelm** [reply]
Probably, your server-level SMTP setup (see pretix.cfg file) is not working correctly
